BestBuyCRUD/SalesDapperRepo.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Dapper;
+using MySql.Data.MySqlClient;
+using System.Linq;
+
+/*namespace BestBuyCRUD
+{
+    /class SalesDapperRepo : ISalesRepository  //adding interface inheritance
```

Comment:
Commented-out code can be removed, and this file can be deleted if it is not being used